bitnami/postgresql/templates/backup/cronjob.yaml

```
@@ -50,7 +50,7 @@ spec:
             {{- if .Values.commonLabels }}
             {{- include "common.tplvalues.render" ( dict "value" .Values.commonLabels "context" $ ) | nindent 12 }}
             {{- end }}
-          {{- if or .Values.annotations .Values.commonAnnotations }}
+          {{- if or .Values.backup.cronjon.annotations .Values.commonAnnotations }}
```

Hi @jcpunk,

There is a typo here:
```suggestion
          {{- if or .Values.backup.cronjob.annotations .Values.commonAnnotations }}